test/ext/KHR_node_visibility/assets.json

```diff
@@ -0,0 +1,10 @@
+{
+    "visibility": {
+        "name": "node.visibility",
```

Comment:
This part should be
```
    "node": {
        "name": "node.KHR_node_visibility",
```
and the directory with test files should be called `data/node` instead of `data/visibility`.